src/dtls/mod.rs

```diff
@@ -317,6 +321,184 @@ impl Dtls {
     }
 }
 
+/// Keying material used as master key for SRTP.
+pub(crate) struct KeyingMaterial(Vec<u8>);
+
+impl KeyingMaterial {
+    #[cfg(any(test, feature = "_internal_test_exports"))]
+    pub(crate) fn new(m: &[u8]) -> Self {
+        KeyingMaterial(m.into())
+    }
+}
+
+impl Deref for KeyingMaterial {
+    type Target = [u8];
+
+    fn deref(&self) -> &Self::Target {
+        &self.0
+    }
+}
+
+impl std::fmt::Debug for KeyingMaterial {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "KeyingMaterial")
+    }
+}
+
+#[derive(Debug, Clone)]
+pub(crate) enum Certificate {
+    #[cfg(feature = "openssl")]
+    Openssl(ossl::DtlsCert),
+}
```

Comment:
I'm in two minds about calling this `Certificate` as long as it's hard tied to DTLS. I guess there's nowhere else in WebRTC we have certificates? 🤔 

I would feel more comfortable staying with `DtlsCert` on the top level and rename the other one `OsslCert` or something.

Reply:
I've renamed it to `DtlsCert`. I've left the `ossl::DtlsCert` name because we can just fully-qualify it to differentiate. The enum is currently not exposed so users only see `ossl::DtlsCert`.